VocaDbWeb/wwwroot/Content/Styles/ExtLinks.less

```diff
@@ -114,14 +147,27 @@ a.extLink {
 		.extLink(url("/Content/ExtIcons/itemtaobao.png"));
 	}
 
+	&[href^="https://www.tanocstore.net/"], &[href^="http://www.tanocstore.net/"] {
+		.extLink(url("/Content/ExtIcons/tanocstore.png"));
+	}
+
 	&[href^="http://www.toranoana.jp/"], &[href^="https://www.toranoana.jp/"] {
 		.extLink(url("/Content/ExtIcons/tora.png"));
 	}
 
+	&[href*="https://ec.toranoana.shop/"], &[href*="http://ec.toranoana.shop/"],
+	&[href*="https://ecs.toranoana.shop/"], &[href*="http://ec.toranoana.shop/"] {
+		.extLink(url("/Content/ExtIcons/tora.png"));
+	}
+
 	&[href^="http://www.tower.com/"], &[href^="https://www.tower.com/"] {
 		.extLink(url("/Content/ExtIcons/towerrecords.png"));
 	}
 
+	&[href^="http://www.tower.jp/"], &[href^="https://www.tower.jp/"] {

```

Comment:
`&[href^="http://tower.jp/"]` and `&[href^="https://tower.jp/"]` should also be added.